Jenkinsfile

```diff
@@ -851,6 +851,26 @@ pipeline {
                         */
                     }
                 }
+                stage ('IOF-build') {
+                    agent {
+                        label 'ci_vm1'
+                    }
+                    options {
+                        timeout(time: 60, unit: 'MINUTES')
```

Comment:
Add a `skipDefaultCheckout` here and then do your IOF checkout right in the root of the workspace.